natives.json

```diff
@@ -89980,15 +89980,15 @@
 			"build": "1207"
 		},
 		"0x6AA3DCA2C6F5EB6D": {
-			"name": "_0x6AA3DCA2C6F5EB6D",
+			"name": "_TASK_GET_ITEM_INTERACTION",
```

Comment:
``_GET_ITEM_INTERACTION_FROM_PED`` is more fitting (based on its alphabetical position).